reddit/client.go

```diff
@@ -23,43 +21,7 @@ type clientConfig struct {
 // client executes http Requests and invisibly handles OAuth2 authorization.
 type client interface {
 	Do(*http.Request) ([]byte, error)
-}
-
-type baseClient struct {
-	cli *http.Client
-}
-
-func (b *baseClient) Do(req *http.Request) ([]byte, error) {
```

Comment:
I'm not sure why these functions are moved to appClient?

It would really help for review if the diff was only as large as it needed to be.

Reply:
I did not find any purpose for the baseClient struct. Just some code simplification while I'm at it